mlflow/java/client/src/main/java/org/mlflow/client/samples/QuickStartDriver.java

```diff
@@ -0,0 +1,86 @@
+package org.mlflow.client.samples;
```

Comment:
Should add a doc comment to this

Reply:
I'd also move the examples into examples/java if possible.

Reply:
Hmm, top-level examples with a self-contained pom.xml is a pretty good idea. Let me see if I can make that work in a followup.